_layouts/episode.html

```diff
@@ -13,7 +13,7 @@
             <ol class="breadcrumb hard text-white">
               <li><a href="/media/">Media</a></li>
               <li><a href="/media/podcasts">Podcasts</a></li>
-              <li><a href="{{ podcast.url }}">{{ podcast.title }}</a></li>
+              <li><a data-media-title="{{podcast.title}}" data-media-title="{{podcast.id}}" href="{{ podcast.url }}">{{ podcast.title }}</a></li>
```

Comment:
You have two `data-media-title` attributes here. I'm guessing that is not correct...

Reply:
Fixed.